NOnion/Constants.fs

```diff
@@ -147,8 +150,15 @@ module Constants =
         let internal AuthInputSuffix =
             protoIdStr + "Server" |> Encoding.ASCII.GetBytes
 
+    module HSDirEncryption =
```

Comment:
let's avoid the Hs prefix, put HiddenService instead

Reply:
Fixed